### DISCUSSION TITLE
Group Field

### DISCUSSION THREAD

**Discussion by forzaxhx** [wazuh/wazuh#14129] [Q&A] [answered]
The 'group.keyword' field of the 'wazuh-monitoring' Elasticseach index should be a field of the 'wauz-alerts' index. With this, the user could filter by group, and dont need to exclude agents manually

**Comment by mikykeane** [comment, marked as answer]
Hi @forzaxhx , 

you can achieve this desired behavior by using Agent labels. As mentioned here: https://documentation.wazuh.com/current/user-manual/capabilities/labels.html

These labels, will enrich the events from any group of agents with a field of your choosing. That way it will help you filter through your events just using that field. 

Let me know if this helped. I will be happy to guide you step by step to configure it correctly. 

Best regards, 
Miguel Keane
